### PR TITLE
Use constant seed for pseudo-random entries creation

### LINKED ISSUE #3384: TestController_IngestRangeHandler test flakes out
https://github.com/treeverse/lakeFS/runs/6490112605?check_suite_focus=true failed with the words
```
time="2022-05-18T13:35:00Z" level=info msg="initialized Auth service" func=pkg/auth.NewDBAuthService file="lakeFS/pkg/auth/service.go:226"
time="2022-05-18T13:35:00Z" level=info msg="initialize OpenAPI server" func=pkg/api.Serve file="lakeFS/pkg/api/serve.go:57"
time="2022-05-18T13:35:01Z" level=info msg="schema migrated" func="pkg/db.(*DatabaseMigrator).Migrate" file="lakeFS/pkg/db/migration.go:49" direction=up host="127.0.0.1:44441" method=POST path=/api/v1/setup_lakefs request_id=cd778902-8352-4339-ab9e-25616a63143c service_name=rest_api took=18.81398ms
--- FAIL: TestController_IngestRangeHandler (1.39s)
    --- FAIL: TestController_IngestRangeHandler/successful_ingestion_no_pagination (1.39s)
        controller_test.go:1241: 
            	Error Trace:	controller_test.go:1241
            	Error:      	Not equal: 
            	            	expected: 1000
            	            	actual  : 312
            	Test:       	TestController_IngestRangeHandler/successful_ingestion_no_pagination
FAIL
```

### PR DESCRIPTION
**Pull request by itaiad200**
closes #3384

### CODE REVIEW

File: pkg/catalog/testutils/testutils.go [hunk 1]
```diff
@@ -148,8 +149,14 @@ const (
 
 func (w *FakeWalker) createEntries(count int) {
 	ents := make([]store.ObjectStoreEntry, count)
+
+	// Use same sequence to overcome Graveler ability to create small ranges.
+	// Calling test functions rely on Graveler to not break on the first 1000 entries.
+	// For example, setting "5" here will cause the test to constantly fail.
+	// Fix Bug #3384
+	rand := rand.New(rand.NewSource(6))
```
Comment (nopcoder) [inline pkg/catalog/testutils/testutils.go:157]: variable name with the same package name

**Review by arielshaqed — APPROVED** [review]
This makes a lot of sense, thanks!  Very good catch >:-)

File: pkg/catalog/testutils/testutils.go [hunk 2]
```diff
@@ -148,8 +149,15 @@ const (
 
 func (w *FakeWalker) createEntries(count int) {
 	ents := make([]store.ObjectStoreEntry, count)
+
+	// Use same sequence to overcome Graveler ability to create small ranges.
+	// Calling test functions rely on Graveler to not break on the first 1000 entries.
+	// For example, setting "5" here will cause the test to constantly fail.
+	// Fix Bug #3384
+	//nolint:gosec
+	randSeed := rand.New(rand.NewSource(6))
```
Comment (nopcoder) [inline pkg/catalog/testutils/testutils.go:158]: This produce the same output every run, right?

Comment (nopcoder) [inline pkg/catalog/testutils/testutils.go:158]:
```suggestion
        const randSource = 6
	//nolint:gosec
	randSeed := rand.New(rand.NewSource(randSource))
```

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: pkg/catalog/testutils/testutils.go (hunk 2 above)
Comment (itaiad200) [reply pkg/catalog/testutils/testutils.go:158]: Right

Comment (itaiad200) [reply pkg/catalog/testutils/testutils.go]: Done.

**Review by nopcoder — APPROVED** [review]
lgtm!

**Pull request merged**